profile/README.md

```diff
@@ -0,0 +1,65 @@
+
+## Products:
+
+* [**Hastlayer**](https://hastlayer.com/) turns performance-critical sections of .NET programs into computer chips (FPGA). You can still update your software, but it will be a whole lot faster and more power-efficient.
+
+ <img src="https://user-images.githubusercontent.com/68704777/173390183-8c2e83ae-9ce3-47c4-a62e-4c58d83648e0.png" width="1000" height="250">
+
+
+* [**DotNest**](https://dotnest.com/) We've developed DotNest as a software-as-a-service for Orchard CMS, enabling users to create their websites using Orchard in a flash. Today it runs more than 6000 websites.
+
+
+<img src="https://user-images.githubusercontent.com/68704777/173390495-9c1fa7a2-084c-4a9b-bc6b-c59e183fb784.png" width="400" height="400">
+ 
+## Our services:
+
+### Software development:
+ 
+Our team's passion is developing software with care for clients who need help with their most extraordinary projects in the following frameworks:
+
+- Orchard Core.
+- Orchard CMS.
+- ASP.NET Core.
+- ASP.NET.
+
+### Consulting:
+
+We offer Orchard consulting in both Orchard editions, Orchard Core and Orchard CMS, including website and application development, and code review.
+
+### Training:
+
+Our team of dedicated experts is here to help groups and individuals who want to learn about Orchard or simply broaden their coding knowledge. Our [Orchard Dojo](https://orcharddojo.net/) website contains several training sessions to help beginners become experts.
+
+
+ <img src="https://user-images.githubusercontent.com/68704777/173390286-1f294d11-feba-44c2-a3e2-d0bcdd3f60ff.png" width="1200" height="250">
+
+
+
+**We also contribute a vast amount of code, documentation, and learning resources to the open-source ecosystem. These materials are also available for our partners to use even after we have delivered a successful project.**
+
+## Our team
+Lombiq's team consists of highly skilled experts from diverse backgrounds:
+
+ <img src="https://user-images.githubusercontent.com/68704777/173390541-bf4c0068-611b-4552-8983-99afb10f4acf.jpg" width="1400" height="700">
+
+
+## Open-source and open hearts
+Apart from the software world we also try to help in other communities with our philanthropic activities. Most recently, we've delivered thousands of reusable masks to the Hungarian Charity Service of the Order of Malta during the coronavirus pandemic, as well as help for refugees of the war in Ukraine, and again.
+
+# Would you like to contribute to our projects?
+
+That would be really fantastic! If you've found one of our projects valuable and you'd like to contribute in any way please check out our [guidelines](https://lombiq.com/open-source-guidelines) and go ahead, we'd be very happy to welcome your efforts! Also, consider contributing, in-kind or otherwise, to the projects supported by us.
+
+You can also check our profile in [NuGet](https://www.nuget.org/profiles/Lombiq).
```

Comment:
```suggestion
You can also check our profile on [NuGet](https://www.nuget.org/profiles/Lombiq) with packages for our open-source projects.
```

Reply:
Next time commit such suggestions from under the PR, please, so they can be auto-resolved.